src/components/containers/menu/MenuPanel.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled, { ThemeProvider } from 'styled-components';
-import defaultTheme from '../../theme/defaultTheme';
```

Comment:
Why was this default removed? If provided, you don't always need to explicitly pass a theme prop in if a parent component has a ThemeProvider.